ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.tsx

```diff
@@ -245,20 +245,20 @@ export const policyConfigurationDescriptor: Descriptor[] = [
         label: 'Image registry',
         name: 'Image Registry',
         shortName: 'Image registry',
-        longName: 'Image pulled from registry',
-        negatedName: 'Image not pulled from registry',
+        longName: 'Container registry name is',
+        negatedName: 'Container registry name is not',
```

Comment:
These are the names mentioned in the meeting to follow up if needed after merge why the difference between **Image** registry and **Container** registry.